src/proxy-ui-api/frontend/src/views/Settings/SystemParameters/UploadConfigurationAnchorDialog.vue

```diff
@@ -12,17 +12,23 @@
         data-test="system-parameters-configuration-anchor-upload-button"
```

Comment:
While you're at it you could add data-test to the input-element before this large-button. It's not a part of this PR, but for the common good :)

Reply:
That input element is not visible in the UI so it can't be clicked in the e2e tests. It's part of the file upload "magic".